consul_exporter.go

```diff
@@ -140,6 +146,20 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 		clusterServers, prometheus.GaugeValue, float64(len(peers)),
 	)
 
+	leader, err := e.client.Status().Leader()
+	if err != nil {
+		log.Errorf("Query error is %v", err)
+	}
+	if len(leader) == 0 {
+		ch <- prometheus.MustNewConstMetric(
+			clusterLeader, prometheus.GaugeValue, 0,
+		)
+	} else {
+		ch <- prometheus.MustNewConstMetric(
+			clusterLeader, prometheus.GaugeValue, 1,
```

Comment:
Sorry should have been clear - leader is a string container the IP:Port of the leader.